src/components/atoms/Alert/Alert.js

```diff
@@ -44,15 +44,15 @@ export default class Alert extends HTMLElement {
 
     // alert attributes
 
-    const icon = this.getAttribute('data-icon');
+    const icon = this.getAttribute('icon');
 
-    const iconOrder = this.getAttribute('data-icon-order');
+    const iconOrder = this.getAttribute('icon-order');
 
-    const iconSize = this.getAttribute('data-icon-size');
+    const iconSize = this.getAttribute('icon-size');
 
-    const backgroundColor = this.getAttribute('data-background-color');
+    const backgroundColor = this.getAttribute('background-color');
 
-    const extraClasses = this.getAttribute('data-extra-classes');
+    const extraClasses = this.getAttribute('extra-classes');
```

Comment:
@sreidthomas this is a breaking change and shouldn't be part of this PR.

While these attributes share the same name as the attributes on the button component, these attributes are owned and controlled by the alert component. This PR is focused on updating the button component attributes, not the alert component attributes. 

The place to look in this file for button attributes would be around [line 100 - 106](https://github.com/CityOfDetroit/COD-Design-System/blob/e967a55376002709169ec7081a07d76d3887844e/src/components/atoms/Alert/Alert.js#L100-L106) where the JS DOM API is used to create a button component. If the `data-icon`, `data-icon-size` etc. were used on the button component on those lines, we would change them as part of this PR. However, those attributes are not used on the button in this file, so we can omit any changes to this file entirely.